src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -654,7 +654,18 @@ private static ImmutableArray<RefKind> GetEffectiveArgumentRefKinds(ImmutableArr
                 var receiverNamedType = invokedAsExtensionMethod ?
                                         ((MethodSymbol)methodOrIndexer).Parameters[0].Type.TypeSymbol as NamedTypeSymbol :
                                         methodOrIndexer.ContainingType;
-            isComReceiver = (object)receiverNamedType != null && receiverNamedType.IsComImport;
+                isComReceiver = (object)receiverNamedType != null && receiverNamedType.IsComImport;
+            }
+
+            var @params = methodOrIndexer.GetParameters();
+            if (!@params.IsDefaultOrEmpty)
+            {
+                var thisParam = @params[0];
```

Comment:
Can elements of this array be `null`?

Reply:
Hard to say for sure, but it seems like the symbol would have to be malformed for that to happen.

Reply:
I would look at other uses of this method and see if those pieces of code consider `null` a valid value.

---
In reply to: [224925020](https://github.com/dotnet/roslyn/pull/30481#discussion_r224925020) [](ancestors = 224925020)

Reply:
All usages of `GetParameters()` that I see so far assume the returned array will not contain null elements.